### DISCUSSION TITLE
Typescript support.

### DISCUSSION THREAD

**Discussion by dcruz1990** [emmanuelsw/notiwind#11] [General]
Hello and good work! Notiwind have TS support?

**Comment by emmanuelsw** [comment]
Hello, thank you!
Currently I don't have enough Typescript knowledge to support it right now but all PRs are welcome!

**Comment by tristanlins** [comment]
Add this to your `env.d.ts`:

```td
declare module 'notiwind' {
    import {Plugin} from 'vue'
    const Notifications: Plugin
    export default Notifications
    export function notify(notification: {
        group: string,
        title: string,
        text?: string,
        kind?: 'error' | 'success',
    }, time: number): void
}
```

The `kind` property is an extra property used in my custom layout.
Feel free to add more properties if needed.
